content/post/data-analyst.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Coolest Job in Tech"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["Analytics", "Org Structure"]
+date: 2020-01-05T21:16:09-05:00
+draft: true
+---
+
+Data analysts at early stage, tech-adjacent start ups have the coolest jobs. For junior and mid level folks especially, there’s so much to learn (hopefully an exciting proposition!)  as analysts  wear many different hats. In addition, there are numerous career paths to take once an analyst has cut their teeth for a few years. Due to its broad set of responsibilities, an analyst’s job can be esoteric and often means different things to different people.
```

Comment:
> as analysts  wear many different hats

i think 
> because analysts  wear many different hats

will be clearer

Reply:
I'd change "In addition" to "even better" to emphasize the positive flow, and I'd add a "However" before "Due to..."